plugins/qsi-dock/qsi-dock.quirk

```diff
@@ -0,0 +1,3 @@
+[USB\VID_047D&PID_808D]
```

Comment:
probably worth the discussion; should all QSI stuff be matched or case by case?

Reply:
case by case; i want vendors to implement the BOS descriptors longer term.